.clang-format

```diff
@@ -0,0 +1,42 @@
+# Defines the Chromium style for automatic reformatting.
```

Comment:
I'll make a note here that this is copied from src/.clang-format unless anyone knows if there's a way to also use the parent config

Reply:
will probably just end up scripting this to either add the options on the command line or copy the file and append